doc/faq.rst

```diff
@@ -43,44 +45,53 @@ details:
 * :option:`CONFIG_UOS_RAM_SIZE`
 * :option:`CONFIG_HV_RAM_SIZE`
 
-For example, if memory is 32G, setup ``PLATFORM_RAM_SIZE`` = 32G
-
-::
-
-  config PLATFORM_RAM_SIZE
-        hex "Size of the physical platform RAM"
-        default 0x200000000 if PLATFORM_SBL
-        default 0x800000000 if PLATFORM_UEFI
-
-Setup ``SOS_RAM_SIZE`` = 32G too (The SOS will have the whole resource)
-
-::
-
-  config SOS_RAM_SIZE
-        hex "Size of the Service OS (SOS) RAM"
-        default 0x200000000 if PLATFORM_SBL
-        default 0x800000000 if PLATFORM_UEFI
-
-Setup ``UOS_RAM_SIZE`` to what you need, for example,  16G
-
-::
-
-  config UOS_RAM_SIZE
-        hex "Size of the User OS (UOS) RAM"
-        default 0x100000000 if PLATFORM_SBL
-        default 0x400000000 if PLATFORM_UEFI
-
-Setup ``HV_RAM_SIZE`` (we will reserve memory for guest EPT paging
-table), if you setup 32G (default 16G), you must enlarge it with
-(32G-16G)/2M pages (where pages are 4K). The example below is after
-HV_RAM_SIZE is changed to 240M
-
-::
-
-  config HV_RAM_SIZE
-    hex "Size of the RAM region used by the hypervisor"
-    default 0x07800000 if PLATFORM_SBL
-    default 0x0f000000 if PLATFORM_UEFI
+For example, if the NUC's physical memory size is 32G, you may follow these steps
+to make the new uefi ACRN hypervisor, and then deploy it onto the NUC board to boot
+ACRN Service VM with the 32G memory size.
+
+#. Modify the ``hypervisor/arch/x86/Kconfig`` from the ACRN source on host machine.
```

Comment:
We should really avoid having people modify the `Kconfig` file directly. This should instead be done using `make menuconfig`